adal/src/Directory.Build.targets

```diff
@@ -1,46 +0,0 @@
-<Project>
```

Comment:
Not seem to be used / imported anywhere - https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/search?q=Directory.Build.targets&unscoped_q=Directory.Build.targets